controllers/trips.js

```diff
@@ -0,0 +1,83 @@
+const User = require("../models/User");
+const Trip = require("../models/Trip");
+const convertDate = require("../utils/utils");
+
+
+//render the index page
+const indexPage = (req, res) => {
+    res.redirect("/trips");
+}
+
+// GET /jobs (display all the job listings belonging to this user)
+const getAllTrips = async (req, res) => {
```

Comment:
Is there a reason the naming here is for trips instead of jobs?

Reply:
yep, basically, my app is not about jobs ;) however I just copied the list of routes from the lesson descriptions and didn't fix the name accordingly